src/Backend/Form/Type/MetaType.php

```diff
@@ -105,11 +169,11 @@ function ($SEOIndex) {
                 SEOIndex::POSSIBLE_VALUES
             ),
             'choice_value' => function (SEOIndex $SEOIndex = null) {
-                return (string) $SEOIndex;
+                return (string)$SEOIndex;
             },
             'choice_label' => function ($SEOIndex) {
                 if ($SEOIndex->isNone()) {
-                    return 'lbl.' . ucfirst($SEOIndex);
```

Comment:
Why remove PSR compliant styles?

Reply:
fixed e7b12fc487faf1f398a7bf31356d5e48f551a271

Reply:
There are some more in the full changes